network/network_test.go

```diff
@@ -186,8 +189,8 @@ func TestNetwork(t *testing.T) {
 	confM.RelayAddrs = relayAddrs
 	confM.Bootstrap.Addresses = bootstrapAddresses
 	confM.Listens = []string{
-		"/ip4/0.0.0.0/tcp/0",
-		"/ip6/::/tcp/0",
+		"/ip4/127.0.0.1/tcp/0",
```

Comment:
I think if we use 127.0.0.1 for tests it perfectly works on shell pipeline runners but in docker pipeline runners it would be better to use 0.0.0.0 which work's like 127.0.0.1 in shell runners.

Reply:
Same as above ☝️